<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-BHR3PSD" failures="1" tests="6" name="EdgeTest" time="36.004" errors="0" timestamp="2024-02-28T15:55:37 IST">
  <testcase classname="testNgPack.VerifyForgotPasswordFunctionality" name="verifyTextOnForgotPasswordPage" time="0.161"/>
  <testcase classname="testNgPack.VerifyLoginAndCreteNewAccFunctionality" name="verifyLoginFunctionality" time="4.476"/>
  <testcase classname="testNgPack.VerifyForgotPasswordFunctionality" name="verifyCancelButtonOnForgotPasswordPage" time="1.165"/>
  <testcase classname="testNgPack.VerifyForgotPasswordFunctionality" name="verifySearchButtonFunctionalityWithWrongId" time="3.413"/>
  <testcase classname="testNgPack.VerifyForgotPasswordFunctionality" name="verifySearchButtonFunctionalityWithCorrectId" time="3.558">
    <failure type="java.lang.AssertionError" message="expected [Log in to Facebook..] but found [Log in to Facebook]">
      <![CDATA[java.lang.AssertionError: expected [Log in to Facebook..] but found [Log in to Facebook]
at testNgPack.VerifyForgotPasswordFunctionality.verifySearchButtonFunctionalityWithCorrectId(VerifyForgotPasswordFunctionality.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- verifySearchButtonFunctionalityWithCorrectId -->
  <testcase classname="testNgPack.VerifyLoginAndCreteNewAccFunctionality" name="creatNewAccFormFunctionality" time="18.237"/>
</testsuite> <!-- EdgeTest -->
